lib/CloudConvert.ts

```diff
@@ -70,7 +70,10 @@ export default class CloudConvert {
                     ? 'https://socketio.sandbox.cloudconvert.com'
                     : 'https://socketio.cloudconvert.com',
                 {
-                    transports: ['websocket']
+                    transports: ['websocket'],
+                    reconnection: true,
```

Comment:
Why need to set `true` here? Default should be already true: https://github.com/socketio/socket.io/blob/main/packages/socket.io-client/lib/manager.ts#L43